config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  resource :user
+
+  resource :user, except: [:update]
   resource :admin
+  patch '/user', to: 'users#update', as: :update_user
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.